### PR TITLE
Working on issue#1721 

### PR DESCRIPTION
**Pull request by RuneAkko**
Remain ineffectual assignments of `iotex-core/action/protocol/account/transfer_test.go` to fix. Draft pull request in case anyone doing repetitive work.

### CODE REVIEW

**Comment by RuneAkko** [comment]
I find the inffectual assignment of transfer_test.go has been fixed by Ko, so this Pr are finished.

File: blockchain/blockchain_test.go [hunk 1]
```diff
@@ -714,7 +714,7 @@ func TestConstantinople(t *testing.T) {
 		total, err := indexer.GetActionCountByAddress(addr27)
 		require.NoError(err)
 		require.EqualValues(7, total)
-		actions, err := indexer.GetActionsByAddress(addr27, 0, total)
+		actions, _ := indexer.GetActionsByAddress(addr27, 0, total)
```
Comment (CoderZhi) [inline blockchain/blockchain_test.go]: require.NoError(err)

File: blockchain/blockdao/indexbuilder_test.go [hunk 2]
```diff
@@ -85,7 +85,7 @@ func TestIndexer(t *testing.T) {
 		startHeight, err := ib.indexer.GetBlockchainHeight()
 		require.NoError(err)
 		require.EqualValues(0, startHeight)
-		tipHeight, err := dao.GetTipHeight()
+		tipHeight, _ := dao.GetTipHeight()
```
Comment (CoderZhi) [inline blockchain/blockdao/indexbuilder_test.go]: require.NoError(err)

File: db/range_index_test.go [hunk 3]
```diff
@@ -224,7 +223,7 @@ func TestRangeIndex2(t *testing.T) {
 		require.Equal(v, NotExist)
 	}
 	for i := uint64(66); i < 70; i++ {
-		v, err = index.Get(i)
+		v, _ = index.Get(i)
```
Comment (CoderZhi) [inline db/range_index_test.go]: require.NoError(err)

**Comment by RuneAkko** [comment]
> please address my comments and clean up your "commits" (20 commits)

Sorry, i didn't notice this pr is still open, when my pr#1682 is merged, I will address  your comments, because I work both pr on my fork-master branch.

**Pull request closed without merging**